### PR TITLE
Seed script

### PR DESCRIPTION
**Pull request by Zio-4**
Created a seed script for default data: users, menu categories, and menu items. Added a new menu category: cocktails.
Also made a change in the package.json file to be able to run the seed script and installed dependencies for seeding.

### CODE REVIEW

File: backEnd/prisma/schema.prisma [hunk 1]
```diff
@@ -25,7 +24,6 @@ model MenuItem {
   description  String
   calorieCount Int
   imageUrl     String
-
```
Comment (RParkerM) [inline backEnd/prisma/schema.prisma:28]: Is there a reason to remove the blank lines here? They were originally added to separate the FKs(foreign keys) from the standard columns, to help make this more readable.

Thanks for removing the commented out code!

File: backEnd/prisma/seed.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+async function main() {
+    // users
+    await Promise.all(defaultUsers.map( async (user) => {
+        return prisma.user.create({
+            data: {
+                username: user.username,
+                firstName: user.firstName,
+                lastName: user.lastName,
+                password: user.password,
```
Comment (RParkerM) [inline backEnd/prisma/seed.ts:12]: Right now, this stores the plain text password in the database. We will want to use the same bcrypt hash mechanism that we use in our signup route so we can login with the passwords set here.

**Review by RParkerM — CHANGES_REQUESTED** [review]
Just need to fix the passwords being stored in the database to be bcrypt hashes rather than plain text, then we're good to go!

File: backEnd/prisma/schema.prisma (hunk 1 above)
Comment (Zio-4) [reply backEnd/prisma/schema.prisma:28]: No reason, I guess I just find it more readable that way. I'll add them back.

File: backEnd/prisma/seed.ts (hunk 2 above)
Comment (Zio-4) [reply backEnd/prisma/seed.ts:12]: Whoops, can't believe I forgot that.

**Pull request merged**